### ISSUE TITLE
integrity check broken in 0.2.4

### ISSUE THREAD

**Issue by robelix** [SecSign/SecSign-ID-Nextcloud-Plugin#6]
the whole .git folder is listed in appinfo/signature.json in the 0.2.4 package
https://github.com/SecSign/SecSign-ID-Nextcloud-Plugin/raw/master/build/artifacts/appstore/secsignid.tar.gz

so nextcloud (16.0.5) shows a long list of missing files:
```
Results
=======
- secsignid
	- FILE_MISSING
		- .git/COMMIT_EDITMSG
		- .git/FETCH_HEAD
		- .git/HEAD
		- .git/ORIG_HEAD
		- .git/config
		- .git/description
		- .git/hooks/applypatch-msg.sample
		- .git/hooks/commit-msg.sample
		- .git/hooks/fsmonitor-watchman.sample
		- .git/hooks/post-update.sample
		- .git/hooks/pre-applypatch.sample
		- .git/hooks/pre-commit.sample
		- .git/hooks/pre-push.sample
		- .git/hooks/pre-rebase.sample
		- .git/hooks/pre-receive.sample
		- .git/hooks/prepare-commit-msg.sample
		- .git/hooks/update.sample
		- .git/index
		- .git/info/exclude
		- .git/logs/HEAD
		- .git/logs/refs/heads/2fa-setup
		- .git/logs/refs/heads/develop
		- .git/logs/refs/heads/master
		- .git/logs/refs/remotes/github/develop
		- .git/logs/refs/remotes/github/master
		- .git/logs/refs/remotes/origin/HEAD
		- .git/logs/refs/remotes/origin/develop
		- .git/logs/refs/remotes/origin/master
		- .git/logs/refs/stash
		- .git/objects/00/5b2b762d8632d891e2610d5a4978ed06ceaaa1
		- .git/objects/00/762e874011a1fabe3430aa091c87872af8cf4c
		- .git/objects/01/4c8918411996dd1a2def61f442580ed2aee7f6
[...]
```

**Comment by secsignid** [comment]
Hi robelix

we will fix that issue as soon as possible. So thank your very much for that hint.

Greetings
The SecSign Team

**Comment by secsignid** [comment]
Hi,
This issue is fixed in versions 0.2.5 for NextCloud < 17 and 0.3.0 for NextCloud 17. Thank you for bringing this to our attention. If you continue having issues, feel free to contact us here or at [support@secsign.com](mailto:support@secsign.com).

Greetings,
The SecSign Team
